index.html

```diff
@@ -258,4 +258,4 @@ <h4 class="text-center subtitles"><span>The JSON representation</span></h4>
     );
 </script>
 </body>
-</html>
```

Comment:
Looks like github's online editor added this empty line to fix the 'No new line at end of file'. Not sure if this change would harm in any way 😄

Reply:
It's OK. Sorry I just noticed I don't have merge permission on this project :disappointed: